vcf-dumper/vcf-dumper-ws/src/main/java/uk/ac/ebi/eva/vcfdump/server/VcfDumperWSServer.java

```diff
@@ -96,7 +96,7 @@ private StreamingResponseBody getStreamingResponseBody(String species, String db
             public void writeTo(OutputStream outputStream) throws IOException, WebApplicationException {
                 VariantExporterController controller;
                 try {
-                    controller = new VariantExporterController(species, dbName, studies, outputStream, evaProperties,
```

Comment:
The species param is not used anymore in the getStreamingResponseBody method and can be removed.